libbeat/outputs/kafka/client.go

```diff
@@ -38,9 +38,9 @@ type msgRef struct {
 }
 
 var (
-	ackedEvents            = expvar.NewInt("libbeat.kafka.published_and_acked_events")
-	eventsNotAcked         = expvar.NewInt("libbeat.kafka.published_but_not_acked_events")
-	publishEventsCallCount = expvar.NewInt("libbeat.kafka.call_count.PublishEvents")
+	ackedEvents            = monitoring.NewInt(outputs.Metrics, "kafka.published_and_acked_events")
```

Comment:
```
kafka.published.events.acked
kafka.published.events.not_acked
```

The last one is kind of a strange one as it counts internal function calls. Do we need this metric or what does it say exactly? Perhaps `kafka.published.events.call`?